examples/platform/posix/radio.cpp

```diff
@@ -260,6 +260,11 @@ int8_t otPlatRadioGetNoiseFloor()
     return 0;
 }
 
+uint32_t otPlatRadioGetCapabilities()
```

Comment:
Nitpick: I'd prefer the API call use the same shorthand as the constants, so either have the API be `otPlatRadioGetCaps()`, or have the constant be `kRadioCapabilityAckTimeout`.

I think using the shorthand for both would be better than using the full word for both because the shorthand would likely be less confusing for [EFL](https://en.wikipedia.org/wiki/English_as_a_second_or_foreign_language) engineers, since it uses the more common `-s` suffix for pluality instead of `-ity` vs `-ies`.

Not a huge issue.